packages/story-editor/src/elements/media/videoImage.js

```diff
@@ -28,23 +28,25 @@ const Video = styled.video`
   object-fit: cover;
 `;
 
-function VideoImage({ alt, ...attrs }) {
+function VideoImage({ attrs }) {
```

Comment:
Can you elaborate on the changes to this compobent?

Reply:
by separating `alt` from `attrs` it's not getting the `alt` set, just the `title` which was giving us some proptype noise. By doing it this way we're still giving the title the `alt` value per usual but allowing `alt` to also be set through spreading the rest of the attrs.

Reply:
But there is no `attrs` prop.

This is how the component is used:

https://github.com/google/web-stories-wp/blob/8464d30ce8ad02d85ffcd5759217920f4c90a5b6/packages/story-editor/src/elements/video/icon.js#L40

The only reason the prop types warning went away with this change is because the newly-added `attrs` prop isn't marked as required. But `attrs` does not exist, so `src` and `alt` are now always `undefined` in that component.

In short, this _creates_ a bug instead of _fixing_ one.

The correct fix would be to update fixture data so resources always contain an `alt`.

I believe #9944 fixes this, so let's defer prop types cleanup to that PR and undo this bug here.

Reply:
sure, will undo